src/lib/components/Feedback/index.js

```diff
@@ -6,18 +6,51 @@ import {BaseElement} from '../BaseElement';
 import {trackEvent} from '../../analytics';
 import './_styles.scss';
 
+/** @types {WebFeedback.Question[]} */
+const questions = [
```

Comment:
You might add a comment explaining that these are default questions that will _always_ be displayed. You could also consider renaming this to `defaultQuestions`.